Blizzard.IO.RabbitMQ.Tests/NetqRabbitConsumerTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Blizzard.IO.Core;
+using Blizzard.IO.RabbitMQ.Entities;
+using EasyNetQ;
+using EasyNetQ.Topology;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+
+namespace Blizzard.IO.RabbitMQ.Tests
+{
+    [TestFixture]
+    public class NetqRabbitConsumerTests
```

Comment:
You are not testing the `IAbstractDeserailzier` options

Reply:
I didn't test the regular `IDeserializer` either, there's no really a good way to check this class.

Reply:
those deserializers are not classes they are interfaces and you need to verify that your class is using the right deserializer the user has chose

Reply:
f